self-run-agents/ami-build/agent-setup.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -e
+
+echo 'debconf debconf/frontend select Noninteractive' | sudo debconf-set-selections
+
+ARCH=$(dpkg --print-architecture)
+
+sudo apt-get update
+sudo apt-get install -y apt-transport-https ca-certificates gnupg-agent software-properties-common wget
+
+wget -q -O - https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+sudo apt-key adv --list-public-keys --with-fingerprint --with-colons 0EBFCD88 2>/dev/null | grep 'fpr' | head -n1 | grep '9DC858229FC7DD38854AE2D88D81803C0EBFCD88'
+sudo add-apt-repository -y "deb [arch=${ARCH}] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+sudo apt-add-repository -y ppa:git-core/ppa
+
+sudo apt-get update
+sudo apt-get install -y docker-ce docker-ce-cli containerd.io libunwind8 libcurl3 git awscli jq
+
+sudo mkdir -p /etc/docker
+echo '{
```

Comment:
What's this for?

Reply:
https://github.com/envoyproxy/envoy/blob/master/.azure-pipelines/bazel.yml#L35-L43